.golangci.yaml

```diff
@@ -141,8 +147,9 @@ issues:
     # Exclude some linters from running on tests files.
     - path: _test\.go
       linters:
-        - gocyclo
-        - dupl
         - gosec
-        - gochecknoglobals
-        - goerr113
+    - path: envtest/environment.go
```

Comment:
where is this located?

Reply:
it's now in `internal` package

Reply:
ah yes so it has to be: `internal/envtest/environment.go`

Reply:
fixed :)